app/policies/pending_profile_policy.rb

```diff
@@ -1,10 +1,9 @@
 # frozen_string_literal: true
 
 class PendingProfilePolicy
-  def initialize(user:, resolved_authn_context_result:, biometric_comparison_requested:)
+  def initialize(user:, resolved_authn_context_result:)
```

Comment:
i don't think we need this `biometric_comparison_requested` parameter, as the `biometric_comparison_requested?` method is looking first at the `resolved_authn_context_result`  object

Reply:
Need to update `app/controllers/concerns/verify_profile_concern.rb#L21` that consumes `PendingProfilePolicy`

Reply:
Fixing `VerifyProfileConcern` should fix the majority of failing tests, too.

Reply:
thank you! fixed in https://github.com/18F/identity-idp/pull/11007/commits/dce21570ba4cd8f930ed51e029d6d2aac132101d